tasks/vagrant.rb

```diff
@@ -100,15 +120,19 @@ def tear_down(node_name, inventory_location)
 end
 
 params = JSON.parse(STDIN.read)
+puts params
```

Comment:
outputting to stderr /stdout or cli, affects what bolt thinks is a successful task. if we want this info it should be in the result in line 137

Reply:
Whoops, this is leftover debugging.

Reply:
@tphoney fixed.